pkg/cmd/cli/cmd/configmap.go

```diff
@@ -47,6 +52,24 @@ func (o *CmdOptions) RunCmdConfigMap(out io.Writer, cmd *cobra.Command, args []s
 	if err != nil {
 		return err
 	}
-	PrintOutput(o.Output, cmap)
+	if cmap != nil && o.Directory != "" {
+		for k, v := range cmap.Data {
```

Comment:
i think it would be nice to have a check if the directory exists, just so we can have a better error message.